geopandas/io/sql.py

```diff
@@ -30,10 +30,27 @@ def read_postgis(
         Active connection to the database to query.
     geom_col : string, default 'geom'
         column name to convert to shapely geometries
-    crs : dict or str, optional
-        CRS to use for the returned GeoDataFrame; if not set, tries to
-        determine CRS from the SRID associated with the first geometry in
-        the database, and assigns that to all geometries.
+    crs : multiple types allowed, optional
+        Coordinate Reference System to use for the returned GeoDataFrame;
+        can accept anything accepted by pyproj.CRS.from_user_input()
+        
+            The following types are accepted:
+            CRS WKT string
+            An authority string (i.e. "epsg:4326")
+            An EPSG integer code (i.e. 4326)
+            A pyproj.CRS
+            An object with a to_wkt method.
+            PROJ string
+            Dictionary of PROJ parameters
+            PROJ keyword arguments for parameters
+            JSON string with PROJ parameters
+            
+            For reference, a few very common projections and their EPSG codes:
+            WGS84 Latitude/Longitude: "EPSG:4326"
+            UTM Zones (North): "EPSG:32633"
+            UTM Zones (South): "EPSG:32733"
+            
+            See geopandas/doc/source/projections.rst for more info on CRS
```

Comment:
```suggestion
    crs : pyproj.CRS, optional
          CRS to use for the returned GeoDataFrame. The value can be anything accepted
          by :meth:`pyproj.CRS.from_user_input() <pyproj.crs.CRS.from_user_input>`,
          such as an authority string (eg "EPSG:4326") or a WKT string.
          If not set, tries to determine CRS from the SRID associated with the 
          first geometry in the database, and assigns that to all geometries.
```

I would keep the docstring in line with other (like [set_geometry](https://github.com/geopandas/geopandas/blob/078062d303e401aaa5e37e04c0e7c3ce188920fe/geopandas/geodataframe.py#L130)). And we need to mention SRID here.